### PR TITLE
Windows Format Paths added (commented)

### PR DESCRIPTION
**Pull request by MarcosVillacanas**
### Description
Examples paths were specified for Linux Users.
Windows users needed to translate them to Windows format, which is as easy as tedious.

### Status
Completed.

### Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or new feature that would cause existing functionality to change)
- [ ] New tests added to cover the changes
- [ ] Docstrings/Documentation updated

### CODE REVIEW

**Comment by Nic-Ma** [comment]
Hi @MarcosVillacanas ,

Thanks for your contribution here.
I suggest using below logic to remove duplicated lines:
```py
import platform

separator = "\" if "Windows" in platform.system() else "/"
images = [
    f"workspace{separator}data{separator}medical{separator}ixi{separator}IXI-T1{separator}IXI607-Guys-1097-T1.nii.gz",
    f"workspace{separator}data{separator}medical{separator}ixi{separator}IXI-T1{separator}IXI175-HH-1570-T1.nii.gz",
    f"workspace{separator}data{separator}medical{separator}ixi{separator}IXI-T1{separator}IXI385-HH-2078-T1.nii.gz",
    ... ...
]
```

What do you think?

Thanks.

**Comment by MarcosVillacanas** [comment]
I didn't think about that option, of course it is better :)
I will update it in a few hours

**Comment by wyli** [comment]
could also consider using `os.sep` or `os.path.join`

**Comment by Nic-Ma** [comment]
> could also consider using `os.sep` or `os.path.join`

Yes, maybe `os.sep` is simpler:
```py
os.sep.join(["workspace", "data", "medical", "ixi", "IXI-T1", "IXI607-Guys-1097-T1.nii.gz"])
```

Thanks.

**Comment by MarcosVillacanas** [comment]
I have replaced the duplication with what you recommended me, and it works in Windows.
Now it looks cleaner and more structured.

File: examples/classification_3d/densenet_training_array.py [hunk 1]
```diff
@@ -27,27 +28,28 @@ def main():
...
-        "/workspace/data/medical/ixi/IXI-T1/IXI175-HH-1570-T1.nii.gz",
-        "/workspace/data/medical/ixi/IXI-T1/IXI385-HH-2078-T1.nii.gz",
-        "/workspace/data/medical/ixi/IXI-T1/IXI344-Guys-0905-T1.nii.gz",
-        "/workspace/data/medical/ixi/IXI-T1/IXI409-Guys-0960-T1.nii.gz",
-        "/workspace/data/medical/ixi/IXI-T1/IXI584-Guys-1129-T1.nii.gz",
-        "/workspace/data/medical/ixi/IXI-T1/IXI253-HH-1694-T1.nii.gz",
-        "/workspace/data/medical/ixi/IXI-T1/IXI092-HH-1436-T1.nii.gz",
-        "/workspace/data/medical/ixi/IXI-T1/IXI574-IOP-1156-T1.nii.gz",
-        "/workspace/data/medical/ixi/IXI-T1/IXI585-Guys-1130-T1.nii.gz",
+        os.sep.join(["workspace", "data", "medical", "ixi", "IXI-T1", "IXI249-Guys-1072-T1.nii.gz"]),
```
Comment (Nic-Ma) [inline examples/classification_3d/densenet_training_array.py:31]: May I know why you change the images in the list?
Thanks.

Comment (MarcosVillacanas) [reply examples/classification_3d/densenet_training_array.py:32]: There was no reason apart from confusion, I have checked the order again and restablish as it was before. My apologies.

**Pull request merged**